blog/docs/articles/eventing-rabbitmq-announcement.md

```diff
@@ -0,0 +1,62 @@
+# Announcing Eventing RabbitMQ moving to GA
+
+**Author: Gab Satchi, Senior Software Engineer @ VMware**
+
+**Date: 2022-07-26**
+
+We’re excited to announce that [Eventing RabbitMQ](https://github.com/knative-sandbox/eventing-rabbitmq) is now GA (General Availability). With this milestone, the APIs provided will remain stable in future iterations.
+A quick start can be found [here](https://knative.dev/docs/eventing/broker/rabbitmq-broker/).
+
+# Key Features
+
+### Use any RabbitMQ instance
+
+Eventing RabbitMQ works well with RabbitMQ clusters provisioned using the [cluster operator](https://github.com/rabbitmq/cluster-operator) but can also be configured to use RabbitMQ instances provisioned by other means.
+Both broker and source have a `rabbitmqClusterReference` which can either reference a `RabbitMQCluster` created using the cluster operator or a `Secret` containing the credentials to a RabbitMQ instance.
+A full guide can be found [here](https://github.com/knative-sandbox/eventing-rabbitmq/tree/49787466f88b21fe022216a13c8321b334f92dc4/samples/external-cluster).
+
+### FIFO vs Throughput
```

Comment:
Please don't use shortened terms like vs, via, etc. It makes the docs vague and difficult to understand.
Maybe something simpler that properly describes the feature, e.g. "Parallelism and event ordering"?